cpp/src/barretenberg/stdlib/primitives/byte_array/byte_array.fuzzer.hpp

```diff
@@ -929,7 +929,7 @@ extern "C" int LLVMFuzzerInitialize(int* argc, char*** argv)
  */
 extern "C" size_t LLVMFuzzerCustomMutator(uint8_t* Data, size_t Size, size_t MaxSize, unsigned int Seed)
 {
-    using FuzzerClass = ByteArrayFuzzBase<plonk::StandardComposer>;
+    using FuzzerClass = ByteArrayFuzzBase<plonk::StandardPlonkComposer>;
```

Comment:
Presumably with this work we're now set up to fuzz the arithmetization (circuit constructor) not really the "composer" - is this right? So there's no need / its not meaningful to fuzz, e.g. Standard Plonk and Standard Honk separately?